<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1290</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1290</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="CTL_BOX">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(250,250,250);
border-style: solid;
border-color:rgb(100,100,100);
border-width: 3px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="LBL_TITLE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225, 225, 225);
border-style: solid;
border-color:rgb(100,100,100);
border-width: 3px;</string>
    </property>
    <property name="text">
     <string>RF Device Controller</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="DEV_AREA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1250</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
